demos/common/cpp/models/src/associative_embedding_decoder.cpp

```diff
@@ -172,7 +171,7 @@ void adjustAndRefine(std::vector<Pose>& allPoses,
             // Refine
             // Get position with the closest tag value to the pose tag
             cv::Mat diff = cv::abs(aembds - poseTag);
-            diff.convertTo(diff, CV_32S, 1.0, 0.5);
+            diff.convertTo(diff, CV_32S, 1.0, 0.0);
```

Comment:
Isn't `diff = np.abs(_tag[..., 0] - prev_tag) + 0.5` is the equivalent for python? It rounds.

Reply:
Apparently I have described the situation incorrectly. In python we do rounds (by +0.5 and after cast to int). In c++ we do +0.5 and after rounds, because `convertTo` doing round automatically. So, if we want to doing postprocessing as in python (it's right version) we must to use parameters 1.0 and 0.0 for `convertTo`.

Reply:
I didn't know `convertTo()` rounds. Thanks

Reply:
@druzhkov-paul, FYI

Reply:
Didn't know the fact. Thanks!